pylint/checkers/base.py

```diff
@@ -2593,6 +2594,41 @@ def _check_type_x_is_y(self, node, left, operator, right):
         self.add_message("unidiomatic-typecheck", node=node)
 
 
+class ThreadingChecker(_BasicChecker):
```

Comment:
Would it be possible to add a new file called ``threading_checker.py``, please ? This file is becoming too huge and we might have to burst it later.

Reply:
moved